modules/standard/Set.chpl

```diff
@@ -541,9 +549,12 @@ module Set {
     :return: A new set containing the union between `a` and `b`.
     :rtype: `set(?t, ?)`
   */
-  proc |(const ref a: set(?t, ?), const ref b: set(t, ?)): set(t) {
+  proc |(const ref a: set(?t, ?), const ref b: set(t, ?)) {
     var result: set(t, (a.parSafe || b.parSafe));
 
+    // TODO: Split-init causes weird errors, see setCompositionParSafe.chpl
```

Comment:
It would be nice to make this comment more specific. Judging by its .good file, setCompositionParSafe.chpl compiles fine.